lib/screens/query_section/widgets/query_message.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+Widget queryMessage(
```

Comment:
change the name of queryMessage to queryMessageCard that makes more sense

Reply:
Over all nice code.. Everything looks cool!

Reply:
That's not a card, that's a message bubble, so will work like this. For reference, see #52

Reply:
make it queryMessageBubble then queryMessage as a name of a widget makes no sense at all